internal/shared/otlp/otlpmetric/otest/collector.go.tmpl

```diff
@@ -299,6 +300,10 @@ func (c *HTTPCollector) Headers() map[string][]string {
 	return c.headers.Clone()
 }
 
+func (c *HTTPCollector) HostHeader() string {
```

Comment:
This getter doesn't appear to be used anywhere. Why do we need it?

Reply:
Sorry, for the confusion, it was used in tests actually but I lost that code at some point and didn't notice. I will bring tests back and this method will be used there